VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,60 @@
+# Video 3: What is AsyncAPI? 
+ 
+Hello, hello!
+
+Welcome to this space! This is one of the videos to understand what AsyncAPI is. I hope it clears up your doubts and you feel like learning more about the project!
+In this video, we'll talk about:
+ 
+ What is AsyncAPI?
+
+- Definition of AsyncAPI
+- How did it come about?
+- What AsyncAPI brings 
+- How it Works / Working case study 
+
+Let’s go!
+
+## Definition of AsyncAPI
+
+AsyncAPI is an open-source specification that aims to make working with EDA as easy today to work with RESTful APIs. 
+
+The strategy is to continue improving, implementing resources that make it more complete and solid. This has led to the recent merger with the Linux Foundation last year. Hosting the project on a neutral foundation guarantees the long-term sustainability and success of the initiative. 
+
+Another important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs and a reference in the sector worldwide. This partnership allows it to drive the growth of more and better tools to help engineers create and maintain asynchronous APIs while using their usual languages and working environments.
+
+AsyncAPI currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs.
+
+2. Improve the performance of the specification to make it more and more seamless. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members of around 400% over the next two years.
```

Comment:
```suggestion
 ## What is AsyncAPI?
 AsyncAPI is an Open-Source specification under the Linux Foundation that aims to make working with Event-Driven Architectures as easy as RESTful APIs. Being under the Linux Foundation guarantees this initiative's long-term sustainability and success. 

Another important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs worldwide. This partnership allows AsyncAPI to drive the growth of more and better tools that help engineers create and maintain asynchronous APIs, all while maintaining their usual languages and working environments.
 
 AsyncAPI aims to accomplish three main goals:
 1. Unify all API specifications. No one does Event-Driven Architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs.
 2. Improve the performance of the specification to make it seamless. Users should be able to create their first API in minutes without prior knowledge of AsyncAPI.
 3. Reach the first position among the most used API specifications. To do so will require about a 400% growth in community members over the next two years.

```

So this section didn't _really_ end up being about defining AsyncAPI only and even ended up being about giving the whole origin story. 😄 That is why I changed the heading ❤️

Reply:
Sure, finally it was much more than a simple definition. 😅